metagraph-plugin/tests/data/edge2.csv

```diff
@@ -0,0 +1,11 @@
+Source,Destination,Weight
```

Comment:
Do you need a file or can you (ideally) this this as part of the code in an array?

Reply:
Either way is fine. But I used "edge2.csv" for consistency with "edge1.csv" and other graphs in this code.

Reply:
If either is fine, can we please build this in memory for now.